src/commonMain/kotlin/exchange.dydx.abacus/output/Account.kt

```diff
@@ -1791,26 +1792,32 @@ data class TradingRewards(
             Logger.d { "creating TradingRewards\n" }
             data?.let {
                 val total = parser.asDouble(data["total"])
-                val historical = total?.let {
+                val fullHistory = total?.let {
                     createHistoricalTradingRewards(
                         it,
-                        existing?.historical,
-                        parser.asMap(data["historical"]),
+                        existing?.fullHistory,
+                        parser.asMap(data["fullHistory"]),
+                        true,
```

Comment:
can we use named params for booleans? (ideally all params) same with the other call below

Reply:
sure